### ISSUE TITLE
Verify GLA Laptop Inventory

### ISSUE THREAD

**Issue by MattCFry** [CodeYourFuture/CYF-Staff-Tickets#571]
### Background

We need to enter all of our records into the new system as part of [this milestone](https://github.com/CodeYourFuture/CYF-Staff-Tickets/milestone/20) before we can roll it out for global usage.

Many global records need verifying before entering the new system as usable records.

### Acceptance criteria

- [x] (Dara-Emma) Identified those on their laptop inventory who should be contacted first. 
- Dara-Emma note: Most individuals on the GLA laptop inventory who have unreturned laptops are from before Dara-Emma started. I was told these were not priority at the time to chase up. Everyone to be contacted was/is in GLA6
- [x] (Dara-Emma) Sent initial reminder to them about the laptop via Slack.
- [x] (Matt or Dara-Emma) Cohort and Email Identified. 

- [x] (Matt) The people identified should be contacted via mail merge.

A week after the mail merge for a region has been sent, the work in that region can be considered complete on #567. 

### Owner

@mattcfry

### Stakeholder

@emcglackenr1 

### Approver

@MattCFry

**Comment by emcglackenr1** [comment]
Relates to #569

**Comment by MattCFry** [comment]
Dara is adding a verified column to approve them being added to the air table.

**Comment by emcglackenr1** [comment]
Potentially helpful sheet from January 2023: https://docs.google.com/spreadsheets/d/1lFg-dvCAXkrqhXXN3RMO5yPwQnR7yY3RpYCZHf6h-i8/edit?resourcekey#gid=1122805472

It is a potentially out of date though.

**Comment by MattCFry** [comment]
Requested access :)

**Comment by emcglackenr1** [comment]
What is keeping this ticket from progressing:

1. Trainees who haven't confirmed the serial number of their borrowed laptops. Volunteers / staff can talk to trainees present on Sat the 17th Feb to try unblock some of this.

2. GLA-6 trainees who dropped out not responding to contact attempts

3. Waiting for a trainee and volunteer to organise meeting up see below 
- Might be broken. Volunteer to assess when it is returned.  If it is damaged by the trainee, then @MattCFry mentioned he will discuss with @ClaireBickley how to proceed. 
- Matt and volunteer CCed into conversation with trainee (16 Feb 2024 at 13:21) so Matt can keep an eye on the situation, and the volunteer can organise a time to collect it from the trainee
- [Posted about on Slack](https://codeyourfuture.slack.com/archives/C029341FPH6/p1708081531374629)

4. Response from volunteer about laptop return
Former trainee said they returned laptop(5CG7380XDS) to volunteer, but the serial number doesn't match what we have in stock. A laptop with the same model(Elitebook 840 G3) but a different serial number(5CG70822ZV) was returned recently by an unknown person to CYF GLA storage. A volunteer checking stock found it there, so I changed from AWOL to Stock. This might be the laptop that was returned, and there was a mistake in the original records [More details in laptop inventory ](https://docs.google.com/spreadsheets/d/1QeZmhAKSD8k4lyIpWrZvZM3voWKf9rnSpI0WpcJIDLg/edit#gid=484306581)

**Comment by MattCFry** [comment]
Hi, @emcglackenr1. Today, I want to close this and the GLA import ticket.

Can you give us any updates on the points you mentioned above?

We can extract any individual circumstances and create a new ticket so this can move on. Any unresponsive people should be marked as AWOL and delt with separately (like we discussed).

**Comment by emcglackenr1** [comment]
Hi Matt, all the updates are in the spreadsheet. 

You can put this in a new issue. If the records are ready to be uploaded I ticket the checkbox in the column I created. If the checkbox is still unchecked, then they are not verified. 
So, all of the unticked checkboxes from column B can be added to the new issue.

**Comment by MattCFry** [comment]
Thank you ❤️ I think we are done for now. I will close this ticket and the import ticket.

When the time comes, we can figure out AWOL laptops. Keep me updated on any other updates on the spreadsheet.
